### PR TITLE
chore(deps): Remove unused deps

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Attempting to begin taming this wild package.json / yarn.lock.

There is a bunch of stuff in here that's just straight up not even used

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,12 +21,8 @@
     "@sentry/webpack-plugin": "^1.20.0",
     "@types/dompurify": "^2.0.3",
     "@types/node": "^12",
-    "@types/react": "^16.9.46",
-    "@types/react-dom": "^16.9.8",
```
Comment (evanpurkhiser) [inline package.json:25]: For some reason these were pinning to v18 of the types, I had to move them to the resolutions for now. I'll revisit later.

File: package.json [hunk 2]
```diff
@@ -21,12 +21,8 @@
...
     "@types/react-helmet": "^6.1.0",
     "@types/react-select": "^3.0.19",
-    "add": "^2.0.6",
```
Comment (evanpurkhiser) [inline package.json:28]: Probably mistakenly added.

File: package.json [hunk 3]
```diff
@@ -21,12 +21,8 @@
...
-    "algoliasearch": "^4.2.0",
```
Comment (evanpurkhiser) [inline package.json:29]: Our own search package pulls this in on it's own

File: package.json [hunk 4]
```diff
@@ -64,14 +60,12 @@
     "platformicons": "^5.3.5",
     "prism-sentry": "^1.0.2",
     "prismjs": "^1.27.0",
-    "prop-types": "^15.7.2",
```
Comment (evanpurkhiser) [inline package.json:67]: unused

File: package.json [hunk 5]
```diff
@@ -64,14 +60,12 @@
...
     "query-string": "^6.13.1",
     "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
     "react-feather": "^2.0.8",
     "react-helmet": "^5.2.1",
-    "react-instantsearch-dom": "^6.4.0",
```
Comment (evanpurkhiser) [inline package.json:74]: unused, probably leftover from before we had our own search package.

File: package.json [hunk 6]
```diff
@@ -81,24 +75,26 @@
...
     "jsdom/**/ws": "^7.4.6",
+    "normalize-url": "^4.5.1",
+    "postcss": "^7.0.36",
     "set-value": ">=4.0.1",
-    "sharp": ">=0.30.0"
+    "sharp": ">=0.30.0",
+    "ssri": ">=8.0.1",
+    "trim": ">=1.0.1",
+    "trim-newlines": "^4.0.2",
+    "xmlhttprequest-ssl": ">=1.6.2"
```
Comment (evanpurkhiser) [inline package.json:97]: sorted

File: package.json [hunk 7]
```diff
@@ -128,10 +124,7 @@
     "babel-preset-gatsby": "^0.5.8",
     "eslint": "^8.38.0",
     "eslint-config-sentry-docs": "^1.114.0",
-    "est": "^0.2.1-alpha",
```
Comment (evanpurkhiser) [inline package.json:131]: unused

File: package.json [hunk 8]
```diff
@@ -128,10 +124,7 @@
...
-    "gh-pages": "^2.2.0",
```
Comment (evanpurkhiser) [inline package.json:132]: unused, probably left over from a time where we tried deploying this to gh pages before vercel

File: package.json [hunk 9]
```diff
@@ -128,10 +124,7 @@
...
     "http-server": "^0.12.3",
-    "identity-obj-proxy": "^3.0.0",
```
Comment (evanpurkhiser) [inline package.json:134]: unused

File: package.json [hunk 10]
```diff
@@ -140,8 +133,7 @@
     "prettier": "^2.8.7",
     "react-test-renderer": "^16.13.1",
     "react-testing-library": "^8.0.1",
-    "ts-jest": "^26.3.0",
-    "tslint-react": "^5.0.0"
```
Comment (evanpurkhiser) [inline package.json:144]: unused

File: package.json [hunk 11]
```diff
@@ -128,20 +124,15 @@
     "babel-preset-gatsby": "^0.5.8",
     "eslint": "^8.38.0",
     "eslint-config-sentry-docs": "^1.114.0",
-    "est": "^0.2.1-alpha",
-    "gh-pages": "^2.2.0",
     "http-server": "^0.12.3",
-    "identity-obj-proxy": "^3.0.0",
     "jest": "^26.4.2",
     "jest-dom": "^4.0.0",
-    "jest-environment-jsdom-fourteen": "^1.0.1",
```
Comment (evanpurkhiser) [inline package.json:137]: not needed

File: package.json [hunk 12]
```diff
@@ -39,15 +35,12 @@
     "gatsby-image": "^2.4.14",
     "gatsby-plugin-algolia": "^0.11.2",
     "gatsby-plugin-google-gtag": "^2.1.11",
-    "gatsby-plugin-manifest": "^2.4.21",
```
Comment (evanpurkhiser) [inline package.json:42]: unused

File: package.json [hunk 13]
```diff
@@ -39,15 +35,12 @@
...
     "gatsby-plugin-mdx": "^1.2.43",
     "gatsby-plugin-meta-redirect": "^1.1.1",
-    "gatsby-plugin-offline": "^3.2.21",
```
Comment (evanpurkhiser) [inline package.json:45]: unused

File: package.json [hunk 14]
```diff
@@ -39,15 +35,12 @@
...
     "gatsby-plugin-react-helmet": "^3.3.10",
     "gatsby-plugin-sass": "^2.3.12",
     "gatsby-plugin-sharp": "^4.25.1",
     "gatsby-plugin-sitemap": "^5.20.0",
-    "gatsby-plugin-typescript": "^2.4.18",
```
Comment (evanpurkhiser) [inline package.json:50]: Already included with gatsby by default

File: package.json [hunk 15]
```diff
@@ -58,20 +51,17 @@
     "gatsby-transformer-javascript-frontmatter": "^2.3.11",
     "gatsby-transformer-json": "^2.4.11",
     "gatsby-transformer-remark": "^2.8.7",
-    "gatsby-transformer-yaml": "^2.4.8",
```
Comment (evanpurkhiser) [inline package.json:61]: unused

**Review by priscilawebdev — APPROVED** [review]
lgtm...great cleanup!

**Review by k-fish — APPROVED** [review]

**Review by shanamatthews — APPROVED** [review]
This is great, thank you for taking the time to do this Evan!

**Pull request merged**